<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1069</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViewManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/**{
	background: rgb(170, 0, 0);
	font: 10pt &quot;Purisa&quot;;
	color: rgb(255, 255, 0);
}*/

/*QTableView{
    alternate-background-color: blue;
    *//* background: yellow; *//*
	selection-color: blue; *//* selected text color *//*
	selection-background-color: yellow;
}*/
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="0" column="3">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>View history</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3" rowspan="3">
     <widget class="QTableView" name="tableView_history">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QTableView" name="tableView_sessions">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTableView" name="tableView_series">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Series Information</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <item row="1" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string># episodes found</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_nepisodes">
           <property name="text">
            <string>eps</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>First Viewed</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_firstViewed">
           <property name="text">
            <string>eps</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Last Viewed</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_lastViewed">
           <property name="text">
            <string>eps</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Times Viewed:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_timesViewed">
           <property name="text">
            <string>eps</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string>NoName</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Series</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_addSeries">
        <property name="toolTip">
         <string>Add Series</string>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_removeSeries">
        <property name="toolTip">
         <string>Remove Series</string>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_seriesSettings">
        <property name="toolTip">
         <string>Series Settings</string>
        </property>
        <property name="text">
         <string>S</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>View Sessions</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_addViewSession">
        <property name="toolTip">
         <string>Add View Session</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="text">
         <string>A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_removeViewSession">
        <property name="toolTip">
         <string>Remove View Session</string>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Episode List</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_refreshEpisodeList">
        <property name="toolTip">
         <string>Refresh Episode List</string>
        </property>
        <property name="text">
         <string>R</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QTableView" name="tableView_episodes">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1069</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionAdd_Series"/>
    <addaction name="actionAdd_View_Session"/>
   </widget>
   <widget class="QMenu" name="menuEpisode">
    <property name="title">
     <string>&amp;Episode</string>
    </property>
    <addaction name="actionView_Next_Episode"/>
    <addaction name="actionView_Episode"/>
    <addaction name="action_Mark_Next_Episode_Viewed"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuEpisode"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Series">
   <property name="text">
    <string>Add &amp;Series</string>
   </property>
  </action>
  <action name="actionAdd_Directory">
   <property name="text">
    <string>Add &amp;Directory</string>
   </property>
  </action>
  <action name="actionAdd_View_Session">
   <property name="text">
    <string>Add &amp;View Session</string>
   </property>
  </action>
  <action name="actionView_Next_Episode">
   <property name="text">
    <string>View &amp;Next Episode</string>
   </property>
  </action>
  <action name="actionView_Episode">
   <property name="text">
    <string>View &amp;Episode</string>
   </property>
  </action>
  <action name="action_Mark_Next_Episode_Viewed">
   <property name="text">
    <string>&amp;Mark Next Episode Viewed</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
